packages/lesswrong/server/apolloServer.ts

```diff
@@ -166,6 +166,9 @@ export function startWebserver() {
     context: async ({ req, res }: { req: express.Request, res: express.Response }) => {
       const context = await getContextFromReqAndRes(req, res);
       configureSentryScope(context);
+      if (performanceMetricLoggingEnabled.get()) {
```

Comment:
We don't have to - I don't expect any significant performance impact from this; was just default to gating this in case you didn't want the risk of it running on the EA forum straight away (though I don't think the risk of unknown unknowns here is that big).  I'll ungate this one.